corehq/apps/domain/tests/test_views.py

```diff
@@ -122,3 +124,30 @@ def test_exchange_snapshot_view(self):
         self.assertEqual(len(snapshots), 2)
         self.assertEqual(snapshots[0].documentation_file_path, filename)
         self.assertEqual(snapshots[1].documentation_file_path, filename)
+
+
+class BaseAutocompleteTest(TestCase):
```

Comment:
I am loath to change this to a database test, but some templates tags (`maintenance_alert`, for example) query postgres. For some reason this does not cause test failure locally but it does on travis. It is notable that "non-database" tests can query postgres without raising errors if the necessary database entities exist.

Notably, non-database tests raise an error when they hit couch because the couch db object is patched. We should probably do the same for postgres, although I assume that will cause a new batch of test failures.